apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/ChronoLocalDateTimeAssertionSpec.kt

```diff
@@ -1,46 +1,183 @@
     include(ChronoLocalDateTimeSpec)
     include(StringSpec)
+    mapOf(
+        "20200101010101111" to "wrong string format",
+        "20200101010101" to "wrong string format",
+        "202001010101" to "wrong string format",
+        "2020010101" to "wrong string format",
+        "2020-1-01T01:01" to "wrong string format",
+        "2020-01-1T01:01" to "wrong string format",
+        "20-01-01T01:01" to "wrong string format",
+        "2020-01-01T1:01" to "wrong string format",
+        "2020-01-01T01:1" to "wrong string format",
+        "2020-01-01T01:01:1" to "wrong string format",
+        "2020-01-01t01:01:01" to "wrong string format"
+    ).forEach{(value, description) ->
+        run {
+            val now = expect(LocalDateTime.now())
+            describe(description) {
+                it("throws an DateTimeParseException") {
+                    expect {
+                        now.isBefore(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+                it("throws an DateTimeParseException") {
+                    expect {
+                        now.isBeforeOrEqual(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+                it("throws an DateTimeParseException") {
+                    expect {
+                        now.isAfter(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+                it("throws an DateTimeParseException") {
+                    expect {
+                        now.isAfterOrEqual(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+                it("throws an DateTimeParseException") {
+                    expect {
+                        now.isEqual(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+            }
+        }
+    }
+
+    mapOf(
+        "2020-01-01T01:01:01" to "allowed shortcuts",
+        "2020-01-01T01:01" to "allowed shortcuts",
+        "2020-01-01" to "allowed shortcuts"
+    ).forEach{(value, description) ->
+        run {
+            val nowLocalDateTime = LocalDateTime.now()
+            val now = expect(nowLocalDateTime)
+            val before = LocalDateTime.now()
+            describe(description) {
+                it("throws an AtriumError") {
+                    expect {
+                        now.isBefore(value)
+                    }.toThrow<AtriumError> { messageContains("is before: $value") }
+                }
+                it("doesn't throw any Error") {
+                    expect {
+                        now.isBefore(before.format(ISO_DATE_TIME))
+                    }
+                }
+                it("doesn't throw any Error") {
+                    expect {
```

Comment:
remove  `expect{...}` as this creates an Expect of a lambda which is never executed. Alternatively you could use `notToThrow` to verify it (instead of toThrow) but we haven't use this so far and I prefer the shorter version